docs/source/upgrading_your_components.md

```diff
@@ -0,0 +1,208 @@
+* `LEDGERS_BACKUP`: the place in your local filesystem where you want to store the ledger being backed up. As you will see below, each node being backed up will have its own subfolder containing its ledger. You will need to create this folder.
+* `IMAGE_TAG`: The version you are upgrading to. This should have a prefix of `(go env GOARCH)`. For example `export IMAGE_TAG=$(go env GOARCH)-<version you are upgrading to>`.
+
+### Upgrade containers
+
+If you have a native deployment, you should copy your `orderer.yaml` file now. Later, you will need this file to compare against the `orderer.yaml` file that is bundled with the release.
+
+Let’s begin the upgrade process by **bringing down the orderer**:
+
+```
+docker stop $ORDERER_ADDRESS
+```
+
+Once the orderer is down, you'll want to **backup its ledger and MSP**:
+
+```
+docker cp $ORDERER_ADDRESS:/var/hyperledger/production/orderer/ ./$LEDGERS_BACKUP/$ORDERER_ADDRESS
+```
+
+Now **restart the ordering node** with our new image.
+
+For native deployments, this is where you will need to compare the `orderer.yaml` file bundled with the release against the `orderer.yaml` you copied earlier. A tool like `diff` can be helpful here. Once you have copied over the relevant differences, you can start the node:
+
+```
+peer node start
+```
+
+Note: as with the environment variables, the use of `peer` is correct here.
+
+Once the ordering node comes up, verify that it is working correctly by querying the ledger. The ordering node should detect that it has been down and pull any blocks it has missed.
+
+## Upgrade the peers
+
+Peers should, like the ordering nodes, be upgraded in a rolling fashion (one at a time). As mentioned during the ordering node upgrade, ordering nodes and peers may be upgraded in parallel, but for the purposes of this tutorial we’ve separated the processes out. At a high level, we will perform the following steps:
+
+1. Stop the peer.
+2. Back up the peer’s ledger and MSP.
+3. Remove chaincode containers and images.
+4. Restart the peer with latest image.
+5. Verify upgrade completion.
+
+### Environment variables
+
+Export the following environment variables before attempting to upgrade your peers.
+
+* `CORE_PEER_ADDRESS`: the url of your peer. Note that you will need to set this variable for each node.
+* `LEDGERS_BACKUP`: the place in your local filesystem where you want to store the ledger being backed up. As you will see below, each node being backed up will have its own subfolder containing its ledger. You will need to create this folder.
+* `IMAGE_TAG`: the version you are upgrading to.
+
+Repeat this process for each of your peers until every node has been upgraded.
+
+### Upgrade containers
+
+If you have a native deployment, you should copy your `core.yaml` file now. Later, you will need this file to compare against the `core.yaml` file that is bundled with the release.
+
+Let’s **bring down the first peer** with the following command:
+
+```
+docker stop $CORE_PEER_ADDRESS
```

Comment:
@joealewine This should use peer container name instead of $CORE_PEER_ADDRESS. In general, we set CORE_PEER_ADDRESS variable with the value peer0.org1.example.com:7051 to connect to the peer.